extract/omie_extractor.py

```
@@ -0,0 +1,417 @@
+import os
+import pandas as pd
+import pytz
```

Ruff (0.11.9)

`pytz` imported but unused

Remove unused import: `pytz`

(F401)